src/lib/API/api.js

```diff
@@ -244,19 +242,26 @@ class API {
   /**
    * `/verify/facerecognition` post api call
    * @param {any} payload
-   * @param {string} provider
+   * @param {string} enrollmentIdentifier
+   * @param {any} axiosConfig
    */
-  async performFaceVerification(
-    payload: any,
-    provider: string = 'zoom',
-    axiosConfig: any = {}
-  ): Promise<$AxiosXHR<any>> {
+  performFaceVerification(payload: any, enrollmentIdentifier: string, axiosConfig: any = {}): Promise<$AxiosXHR<any>> {
     const { client } = this
-    const endpoint = `/verify/facerecognition/${provider}`
+    const endpoint = `/verify/face/${enrollmentIdentifier.toLowerCase()}`
 
-    const response = await client.post(endpoint, payload, axiosConfig)
+    return client.put(endpoint, payload, axiosConfig)
+  }
+
+  /**
+   * `/verify/facerecognition` post api call
+   * @param {string} enrollmentIdentifier
+   * @param {string} signature
+   */
+  disposeFaceSnapshot(enrollmentIdentifier: string, signature: string): Promise<void> {
+    const { client } = this
+    const endpoint = `/verify/face/${enrollmentIdentifier}?signature=${signature}`
```

Comment:
1. encode enrollmentIdentifier
2. pass signature via `params` property of axios request config
https://github.com/axios/axios#request-config